source-map-uploader-plugin.js

```diff
@@ -17,6 +18,7 @@ class BugsnagSourceMapUploaderPlugin {
     this.appVersion = options.appVersion
     this.overwrite = options.overwrite
     this.endpoint = options.endpoint
+    this.extensions = options.extensions || [ '.js' ]
```

Comment:
What extension is this? Is it the extension of the file used to build the bundle or the bundle extension? Could we make this clearer in the option name? How does jsx sit with this?

Reply:
It's the bundle extension, not the input. It will work with any input file – it doesn't care (`.ts`, `.jsx`, `.coffee` etc.) as long as the output compiles to a `.js` bundle.

@bullmo was also alarmed by this change, so I'm happy to do something to avoid confusion.

I could rename it `bundleExtensions`?

Reply:
@bullmo also made the point that the bundle in react native has the extension `.bundle` or `.jsbundle` – but at the moment this plugin is not used on react native projects (though it conceivably could, especially with tools like `haul` gaining popularity).

Should we add those extensions to the default just in case/to future proof?

Reply:
Although @Cawllec approved, he did also question why it was an "include" directive and not "exclude". Based on the fact there may be extensions we would want in the future that we don't yet know, I'm inclinded to flip this around into an "exclude" list:

`ignoredBundleExtensions: [ '.css' ]`